orttraining/orttraining/test/python/orttraining_ortmodule_tests.py

```diff
@@ -146,7 +146,7 @@ def run_data_sampler_tests(cwd, log):
 def run_hooks_tests(cwd, log):
     log.debug("Running: Data hooks tests")
 
-    command = [sys.executable, "-m", "pytest", "-sv", "orttraining_test_hooks.py"]
+    command = [sys.executable, "-m", "pytest", "-sv", "orttraining_test_ortmodule_hooks.py"]
```

Comment:
nit: strictly speaking, the hooks here is training/utils/hooks, not bound to ortmodule.